process/rewardTransaction/interceptedRewardTransaction_test.go

```diff
@@ -0,0 +1,148 @@
+package rewardTransaction
```

Comment:
we should use rewardTransaction_test instead as package

Reply:
done